part-one-answer.js

```diff
@@ -0,0 +1,17 @@
+function showresult(val){
+    var n1=parseFloat(document.getElementById('num1').value);
+    var n2=parseFloat(document.getElementById('num2').value);
+    var r=n1+n2;
+    /*var c=val;
+    
+    switch(c)
+        {
+        case '1':
+            r=n1+n2;
+            break;
+        default:
+            break;     
+        }*/
+    document.getElementById('result').value=c;
```

Comment:
Unfortunately as you started in on the `switch` (which we will cover tonight) you only half undid it. You're still trying to put `c` on the screen but `c` doesn't exist anymore, your actual result is in `r`. As a result the answer doesn't make it to the screen which cost you some points.